department-of-veterans-affairs/caseflow
# Adding organization

Connects #6276 #6277 

### Description
Added a new organization table that has a polymorphic association with the task table. This means we can assign tasks to an organization instead of just a user.

### Testing Plan
1. Create a task (Had to create a temporary generic task class for our type)
1. Create an organization
1. Ensure we can associate the organization with the task
